_includes/header.html

```diff
@@ -0,0 +1,30 @@
+<header>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-md-12">
+                <div class="nav-section">
+                    <div class="logo">
+                        <a href="{{site.url}}{{ site.baseurl }}"><img src="{{ '/assets/img/logo.svg' | relative_url }}"></a>
+                    </div>
+                    <a href="#" class="mobile-trigger"></a>
+                    <div class="header-lists">
+                        <ul class="menu">
+                            <li><a href="{{site.url}}{{ site.baseurl }}/#research-log">Research Log</a></li>
+                            <li><a href="{{site.url}}{{ site.baseurl }}/#what">What</a></li>
+                            <li><a href="{{site.url}}{{ site.baseurl }}/#who">Who</a></li>
+                            <li><a href="https://forum.vac.dev/" target="_blank">Forum</a></li>
+                            <li><a href="{{site.url}}{{ site.baseurl }}/#specs" target="_blank">Specs</a></li>
```

Comment:
```suggestion
                            <li><a href="https://specs.vac.dev" target="_blank">Specs</a></li>
```

Reply:
+1, alternatively if we can construct url based on parameters set in https://github.com/vacp2p/vacp2p.github.io/pull/2/files#diff-aeb42283af8ef8e9da40ededd3ae2ab2

Reply:
I've changed the url.